gulpfile.babel.js

```diff
@@ -0,0 +1,15 @@
+import gulp from 'gulp';
+import loadPlugins from 'gulp-load-plugins';
+const plugins = loadPlugins({
+  lazy: false,
+});
+
+gulp.task('lint', () =>
```

Comment:
I'll leave to you - my global eslint can't find packages from within the prowac project
`Error: Cannot find module 'eslint-config-airbnb/base'`